thali/thalilogger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var winston = require('winston');
+
+module.exports = function(){
```

Comment:
O.k. this is a question of aesthetics but I honestly don't think this is pretty. It means people have to:

```javascript
var thaliLogger = require('thalilogger');
var loggingObject = thaliLogger();
```

Or the equally inelegant:

```javascript
var logger = require('thalilogger')();
```

Why not just change the code to:
```Javascript
module.exports = return winston;
```

That way logging becomes:

```Javascript
var logger = require('thalilogger');
```

But that is just a general point.